datafusion/core/src/execution/context.rs

```diff
@@ -1295,19 +1295,7 @@ impl SessionConfig {
             map.insert(entry.key, entry.value.unwrap_or_default());
         }
 
-    /// Enables or disables the round robin repartition for increasing parallelism
-    pub fn with_round_robin_repartition(mut self, enabled: bool) -> Self {
-        self.config_options
-            .set_bool(OPT_ENABLE_ROUND_ROBIN_REPARTITION, enabled);
-        self
-    }
-
-    /// Returns true if the physical plan optimizer will try to
-    /// add round robin repartition to increase parallelism to leverage more CPU cores.
-    pub fn round_robin_repartition(&self) -> bool {
-        self.config_options
-            .get_bool(OPT_ENABLE_ROUND_ROBIN_REPARTITION)
-            .unwrap_or_default()
+        map
     }
```

Comment:
These two were added in #4757. They're no longer necessary and so I just opted to remove them